.github/workflows/diagram-cache.yml

```
@@ -0,0 +1,57 @@
+# Copyright DB Netz AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+name: update_capella_diagram_cache
+
+on: push
+
+env:
+  CAPELLA_DOCKER_IMAGES_REVISION: main
+
+  # Can either be no or selected
+  CAPELLA_DOCKER_IMAGES_DROPINS: selected
+
+  # Semantic Capella version (x.x.x) >= 5.0.0
+  # Defaults to 6.0.0 if not defined
+  CAPELLA_VERSION: 6.0.0
+
+jobs:
+  update_capella_diagram_cache:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout code
+          uses: actions/checkout@v3
+        - name: Set up Python 3.11
+          uses: actions/setup-python@v3
+          with:
+            python-version: 3.11
+        - name: Install capellambse
+          run:
+            pip install "capellambse[cli]@git+https://github.com/DSD-DBS/py-capellambse.git@$CAPELLAMBSE_REVISION"
+        - name: Create diagram cache
+          run: |
+            python << EOF
+            import capellambse
+            model = capellambse.MelodyModel(
+                "./coffee-machine-demo.aird", diagram_cache="./diagram_cache"
```

```suggestion
                os.environ["ENTRYPOINT"], diagram_cache="./diagram_cache"
```

We can make use of the entrypoint environment variable.

---

good point ^^